### DISCUSSION TITLE
get data from DynamoDB at Local

### DISCUSSION THREAD

**Discussion by Marbyun** [apache/airflow#36730] [Q&A]
hi!
i'm new here and now have some task to create a airflow job to get data from dynamoDB at my local i setting it by follow [this](https://www.linkedin.com/pulse/setup-local-dynamodb-docker-nosql-workbench-corinne-roosen/).

when i try to get data from the dynamodb its always get error 
```
botocore.exceptions.NoRegionError: You must specify a region.
```

my set connection:
- conn type : Amazon Web Services
- AWS Key ID : #Base on credential give on NoSQL WorkBench (use this on my cmd and can get the tables)
- AWS Secret Key  : #Same for AWS Key ID
- Extra : (this one i  try "region_name": "localhost"

this is my code i use:

```
def get_dynamodb_data_and_insert():
    dynamodb_conn_id = 'Dynamo-local'  
    dynamodb_hook = BaseHook.get_connection(dynamodb_conn_id)
    dynamodb = boto3.resource('dynamodb', endpoint_url=dynamodb_hook.host)
    table = dynamodb.Table('table_name')  
    response = table.scan()
    items = response.get('Items', [])
    df_raw = pd.DataFrame(items)
    ...
```

fyi i setup my airflow on docker too

**Comment by ParitoshBh** [comment]
Refer https://github.com/apache/airflow/issues/13785#issuecomment-774150220
